openmc/lib/core.py

```diff
@@ -170,6 +169,53 @@ def export_properties(filename=None, output=True):
     with quiet_dll(output):
         _dll.openmc_properties_export(filename)
 
+def export_weight_windows(filename=None, output=True):
+    """Export weight windows.
+
+    .. versionadded:: 0.13.3
+
+    Parameters
+    ----------
+    filename : str or None
```

Comment:
If we add ```from openmc.checkvalue import PathLike``` to the top of the file then this could be changed to ```PathLike

```suggestion
    filename : PathLike or None
```